Backend/router/v1/user.js

```diff
@@ -23,6 +25,10 @@ router.all('/', (req, res) => {
   res.send("this is user root");
 });
 
+function password_length_check(length){
```

Comment:
Hmm I remember seeing this code in another PR. If this is something new, try extracting this function out in to a common folder so we don't have to copy+paste this here and there?